manifests/v1beta1/components/ui/rbac.yaml

```diff
@@ -19,6 +19,18 @@ rules:
       - suggestions
     verbs:
       - "*"
+  - apiGroups:
+      - ""
+    resources:
+      - pods
+    verbs:
+      - list
+  - apiGroups:
+      - ""
+    resources:
+      - pods/log
+    verbs:
+      - get
```

Comment:
I think, we should also update the `kubeflow-user-edit` role: https://github.com/kubeflow/katib/blob/master/manifests/v1beta1/installs/katib-with-kubeflow/kubeflow-katib-roles.yaml#L22-L36 with this permission.

cc @kimwnasptd

Reply:
Edited this as well.